MAINTAINERS.md

```diff
@@ -6,7 +6,6 @@ This document contains a list of maintainers in this repo. See [opensearch-proje
 
 | Maintainer              | GitHub ID                                               | Affiliation |
 | ----------------------- | ------------------------------------------------------- | ----------- |
-| Hanguang Zhang          | [zhanghg08](https://github.com/zhanghg08)               | Amazon      |
 | Jack Mazanec            | [jmazanec15](https://github.com/jmazanec15)             | Amazon      |
```

Comment:
Could you add @amitgalitz @sudiptoguha @jackiehanyang @sean-zheng-amazon and remove @wnbts (who isn't member of the org anymore)?